### DISCUSSION TITLE
How do deactivate the default value in the prompt when adding a new remote?

### DISCUSSION THREAD

**Discussion by azzamsa** [NeogitOrg/neogit#1122] [Q&A] [answered]
Hello. 👋🏼

Is it possible to deactivate the default value in the prompt when adding a new remote?
I want just an empty value because I must always delete the entire default value first

![image](https://github.com/NeogitOrg/neogit/assets/17734314/932a0b9b-f085-4d44-8fc8-31ee587e1eb6)

Thanks!

**Comment by CKolkey** [comment, marked as answer]
Yea, so there's not a way to do that currently - I just kinda assumed it would be a helpful thing. I'll put this as an issue if anyone wants to pick it up :)

**Reply by azzamsa** [reply]
Yes, turns out it is. Or I can just use `C-u` as @star-szr  suggested.

**Comment by star-szr** [comment]
In the meantime, you may find ctrl-u useful!
